index.html

```diff
@@ -0,0 +1,441 @@
+          <div class="header-top-side">
+            <!-- Logo section -->
+            <div class="twitter-logo">
+              <a href="#">
+                <img src="./twitter-icon/twitter-bird-icon.svg" alt="" />
+              </a>
+            </div>
+
+            <!-- Menu area -->
+            <div class="left-menu-type">
+              <div class="menu-list-content">
+                <div>
+                  <a href="#">
+                    <img
+                      src="./twitter-icon/twitter-home-icon.svg"
+                      alt=""
+                      class="menu-icon-image"
+                    />
+                    <b>Home</b>
+                  </a>
+                </div>
+              </div>
+
+              <div class="menu-list-content">
+                <div>
+                  <a href="#">
+                    <img
+                      src="./twitter-icon/twitter-explore-icon.svg"
+                      alt=""
+                      class="menu-icon-image"
+                    />
+                    <b>Explore</b>
+                  </a>
+                </div>
+              </div>
+
+              <div class="menu-list-content">
+                <div>
+                  <a href="#">
+                    <img
+                      src="./twitter-icon/twitter-notify-icon.svg"
+                      alt=""
+                      class="menu-icon-image"
+                    />
+                    <b>Notification</b>
+                  </a>
+                </div>
+              </div>
+
+              <div class="menu-list-content">
+                <div>
+                  <a href="#">
+                    <img
+                      src="./twitter-icon/twitter-message-icon.svg"
+                      alt=""
+                      class="menu-icon-image"
+                    />
+                    <b>Messages</b>
+                  </a>
+                </div>
+              </div>
+
+              <div class="menu-list-content">
+                <div>
+                  <a href="#">
+                    <img
+                      src="./twitter-icon/twitter-boomark-icon.svg"
+                      alt=""
+                      class="menu-icon-image"
+                    />
+                    <b>Bookmarks</b>
+                  </a>
+                </div>
+              </div>
+
+              <div class="menu-list-content">
+                <div>
+                  <a href="#">
+                    <img
+                      src="./twitter-icon/twitter-blue-icon.svg"
+                      alt=""
+                      class="menu-icon-image"
+                    />
+                    <b>Twitter Blue</b>
+                  </a>
+                </div>
+              </div>
+
+              <div class="menu-list-content">
+                <div>
+                  <a href="#">
+                    <img
+                      src="./twitter-icon/twitter-profile-icon.svg"
```

Comment:
I believe part of the criteria is to have one of the icons within the html code using the <svg> tags. Good clean referencing of them throughout - but one as an example would be good!